contracts/GSN/bouncers/GSNBouncerSignature.sol

```diff
@@ -22,6 +22,7 @@ contract GSNBouncerSignature is GSNBouncerBase {
      * @dev Sets the trusted signer that is going to be producing signatures to approve relayed calls.
      */
     constructor(address trustedSigner) public {
+        require(trustedSigner != address(0), "Trusted Signer can not be a Zero Address");
```

Comment:
```suggestion
        require(trustedSigner != address(0), "GSNBouncerSignature: trusted signer is the zero address");
```

I've updated the revert message to better reflect the general style of the library.

Reply:
Have we written down this convention anywhere?

Reply:
There are guidelines in the forum, but I don't think we've added anything to the repo, no.